pallets/multisig/src/lib.rs

```diff
@@ -915,6 +946,7 @@ impl<T: Config> Pallet<T> {
         Self::ensure_max_signers(&multisig, signers.len() as u64)?;
 
         Self::base_authorize_signers(ms_did, &multisig, &signers)?;
+        Self::set_invalid_proposals(&multisig);
         Self::deposit_event(Event::MultiSigSignersAuthorized {
```

Comment:
We should only invalidate the proposals when the signers accept the auth, since only then does the number of signers change.  In the `base_accept_multisig_signer` method below.